L/licensecheck/build_tarballs.jl

```diff
@@ -1,7 +1,7 @@
 using BinaryBuilder
 
 name = "licensecheck"
-version = v"0.3.1"
+version = v"0.300.101"
```

Comment:
For the time being let's expand only the patch version?
```suggestion
version = v"0.3.101"
```
Otherwise looks good